agent/semgrep_agent.py

```diff
@@ -22,7 +22,11 @@
 )
 logger = logging.getLogger(__name__)
 
-COMMAND_TIMEOUT = 90
+COMMAND_TIMEOUT = 120
```

Comment:
I think 120s is not enough for very large files, did you test?

Reply:
well before we had 5 seconds, even tho the timeout to the command was 90, semgrep has a default timout per file of 5 seconds. did you?

Reply:
> well before we had 5 seconds, even tho the timeout to the command was 90

I am aware of that, in fact we're having this PR because I mentioned this exact issue

> did you?

This issue assigned to you, not me.

Here is the same configuration timing out on 94 rules

![image](https://github.com/Ostorlab/agent_semgrep/assets/129080649/d7f32296-93dc-44f4-829b-efffd2e7f6b7)

Reply:
The point of increasing the timeout is to expand the range of files that can be processed and not support every possible file.
if there is a timeout value there will always be a file where this happens, that's the point of timeout.